src/wallet/gtest/test_wallet.cpp

```diff
@@ -350,7 +350,7 @@ TEST(wallet_tests, set_invalid_note_addrs_in_cwallettx) {
     EXPECT_THROW(wtx.SetSproutNoteData(noteData), std::logic_error);
 }
 
-TEST(wallet_tests, GetNoteNullifier) {
+TEST(wallet_tests, GetSproutNoteNullifier) {
```

Comment:
We need corresponding tests for Sapling (or, better, some way to share the test code, but that looks difficult).